plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraTypeMapping.java

```diff
@@ -558,6 +628,18 @@ private DataSetup cassandraCreateAndInsert(String tableNamePrefix)
         return new CassandraCreateAndInsertDataSetup(session::execute, tableNamePrefix, server);
     }
 
+    private TestCassandraTable testTable(String namePrefix, List<ColumnDefinition> columnDefinitions, List<String> rowsToInsert)
+    {
+        return new TestCassandraTable(session::execute, server, "tpch", namePrefix, columnDefinitions, rowsToInsert);
+    }
+
+    private void assertCassandraQueryFails(@Language("SQL") String sql, String expectedMessage)
```

Comment:
> `@Language("SQL")`

There's no benefit if you use `String.format()` in the caller side. Could you replace with concatenation or remove this annotation?

Reply:
> Could you replace with concatenation or remove this annotation?

@ebyhr From your perspective, which one do you recommend more? (For me, I would remove this annotation because the changes are minimal)

Reply:
I think replacing with concatenation is better. As you may already know, IntelliJ provides shortcut  for `String.format()` ↔︎ concatenation. 
<img width="901" alt="Screen Shot 2022-04-27 at 16 43 41" src="https://user-images.githubusercontent.com/6237050/165467661-503e2a06-4426-4707-9950-a7b9918c4084.png">

Reply:
Got it, and I'll also fix the same problem with other connectors.

Reply:
Addressed.